.github/ISSUE_TEMPLATE/compatible-certification-request.md

```diff
@@ -0,0 +1,18 @@
+- [ ] Organization Name ("Organization") and, if applicable, URL:<br/>
```

Comment:
The Compatible Certification Request should be filed against the Specification repository, not the microprofile-wg repository.  So, it doesn't really belong here...  Each of the Spec repositories should have this template available for creating CCRs.  We can use this one as a "template for a template", but we wouldn't actually use it here.